### DISCUSSION TITLE
Annotations List Render HTML

### DISCUSSION THREAD

**Discussion by michaelvannevel** [grafana/grafana#38184] [Dashboards, Panels & Visualizations]
**What would you like to be added**: I would like the ability to have HTML rendered in the annotation list.

**Why is this needed**: Its interesting that the marker at the bottom the lines on charts renders HTML but the annotation list does not.  It seems inconsistent.  So my annotations look good on the panels, but when i want to have a dashboard with the annotation list i see all the html characters instead.

**Comment by ivanahuckova** [comment]
This is a community panel so please with any feature request or bug fix, refer there https://github.com/ryantxu/annotations-panel

**Comment by michaelvannevel** [comment]
@ivanahuckova The Annotations List is in Core.  The Title was misleading, I will fix that. The content of the issue was correct.

**Comment by ivanahuckova** [comment]
Oh, sorry, I've misunderstood. I have re-opened the issue and added labels and passed to the related project & team. 🙂

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
